README.md

```diff
@@ -238,9 +212,11 @@ After installing the plugin and syncing your Android Studio project with Gradle,
             // 4. Instantiate an instance of the CodePush runtime, using the right deployment key
             this._codePush = new CodePush("0dsIDongIcoH0mqAmoR0CYb5FhBZNy1w4Bf-l", this, BuildConfig.DEBUG);
 
-            // 5. Add the CodePush package to the list of existing packages
+            // 5. Add the CodePush package to the list of existing packages,
+            // replacing the `new CodePushReactPackage()` added by RNPM
             return Arrays.<ReactPackage>asList(
-                new MainReactPackage(), this._codePush.getReactPackage());
+                new MainReactPackage(), /* REMOVE THIS -> new CodePushReactPackage() */
```

Comment:
Would it be possible to support an API where the package constructor took the deployment key and activity reference, and there was a static method for getting the JS bundle URL? It would be cool if instead of requiring the dev to remove the code added by RNPM and then adding another line, they just simply had to pass two parameters to the constructor that RNPM already injected for them.